src/test/scala/com/github/vitalsoftware/scalaredox/UploadTest.scala

```diff
@@ -0,0 +1,25 @@
+package com.github.vitalsoftware.scalaredox
+
+import java.io.File
+
+import org.specs2.mutable.Specification
+import org.specs2.time.NoTimeConversions
+
+/**
+ * @author andrew.zurn@dexcom.com - 11/1/17.
+ */
+class UploadTest extends Specification with NoTimeConversions with RedoxTest {
+
+  "alter Uploads" should {
+    "post new Uploads" in {
+      val file = new File("/Users/az0916/Downloads/clarity_2017-10-26_220047.pdf")
```

Comment:
@AndrewZurn Maybe add a test file to the `/conf` or `test` directory that is accessible during an `sbt test` run rather than a local file. That way automated tests for Jenkins/Codeship will pass. Right now they fail as they cannot find this local file.

Reply:
Whoops. Totally had a it creating a temporary one to use during testing. Let me get that as well.